resources/linux/code.appdata.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
```

Comment:
If you wanted to split the .appdata.xml change out into a separate PR we can merge this a lot faster.